deployment/docker/vmanomaly/vmanomaly-integration/vmanomaly_guide_dashboard.json

```diff
@@ -40,7 +40,7 @@
           "showLineNumbers": false,
           "showMiniMap": false
         },
-        "content": "If you don't see any data, please wait a few minutes. \n\nYou will see a lot of false positive anomalies when you run the guide for the first time. \nThe prediction must be more accurate if you provide vmanomaly 2w of data.\n\nEvery row represents information for one specific mode. \nThe query for anomaly detection is `quantile by (mode) (0.5, rate(node_cpu_seconds_total[5m]))`\nThis is a median (or 50% quantileof `rate` function over `node_cpu_seconds_total`)",
+        "content": "If you don't see any data, please wait a few minutes. \n\nYou will see a lot of false positive anomalies when you run the guide for the first time. \nThe prediction must be more accurate if you provide vmanomaly 2w of data.\n\nEvery row represents information for one specific mode. \nThe query for anomaly detection is `sum(rate(node_cpu_seconds_total)) by (mode)`\n",
```

Comment:
Let's use better wording here, something like:

> "If you don't observe any data initially, please wait a few minutes for it to appear.
> Upon first running the guide (if there is not enough node_exporter monitoring data collected in your system), you may notice a significant number of false positive anomalies found. The predictions will become more accurate with at least two weeks' (full `fit_window`) worth of data provided to vmanomaly.

> Each row displays information for a distinct mode. The query used for anomaly detection is sum(rate(node_cpu_seconds_total)) by (mode)."

Also, it looks like a discrepancy:
in vmanomaly config we use
```markdown
queries:
    node_cpu_rate: "sum(rate(node_cpu_seconds_total[5m])) by (mode, instance,job)"
```
while in this paragraph we use `sum(rate(node_cpu_seconds_total)) by (mode)` (different group by clause)

Reply:
Changed the text
The discrepancy was made on purpose to simplify the header. But I changed it to an actual query now